packages/cdktf/lib/tokens/token.ts

```diff
@@ -171,6 +171,19 @@ export class Token {
   public static asAny(value: any): IResolvable {
     return Tokenization.isResolvable(value) ? value : new Intrinsic(value);
   }
+
+  /**
+   * Return a Token containing a `null` value
+   * @returns a Token resolving to `null` as understood by Terraform
+   * Note: This is different than `undefined`, `nil`, `None` or similar
```

Comment:
The note is not included in the generated documentaton, is this by choice?

Reply:
Nope, by accident 😅 I'll re-order